plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetGalleryFacetImpl.java

```diff
@@ -425,25 +435,30 @@ public DateTime getLastVerified(final String id, final String version) {
   @Override
   public void setLastVerified(final String id, final String version, final DateTime date) {
```

Comment:
Instead of propagating IOEx, let's just change the method signature to throw it - this is only called from NugetProxyFacet.indicateVerified (which itself throws IOEx).